CHANGELOG.md

```diff
@@ -2,6 +2,19 @@
 
 All notable changes to this project will be documented in this file. The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) and this project adheres to [Semantic Versioning](http://semver.org).
 
+## [v1.1.1](https://github.com/puppetlabs/puppetlabs-hocon/tree/v1.1.1) (2020-03-02)
+
+[Full Changelog](https://github.com/puppetlabs/puppetlabs-hocon/compare/v1.1.0...v1.1.1)
+
+### Fixed
+
+- \[HC-99\] Fix to replace an existing array with an empty array [\#105](https://github.com/puppetlabs/puppetlabs-hocon/pull/105) ([trevor-vaughan](https://github.com/trevor-vaughan))
+
+### UNCATEGORIZED PRS; GO LABEL THEM
```

Comment:
Looks like two of these need labeling - I'm guessing EL8 will be a feature (as it adds support) and the mergeback should be maintenance - then the changelog will need to be regenerated.